workflows/qc/quality-check-standard.wdl

```diff
@@ -402,21 +409,21 @@ task parse_input {
     meta {
         description: "Parses and validates the `quality_check` workflow's provided inputs"
         outputs: {
-            check: "Dummy output to indicate success and to enable call-caching",
+            check: "Dummy output to indicate success and to enable call-caching",  # TODO: is this still needed with labels?
             labels: "An array of labels to use on the result coverage files associated with each coverage BED"
         }
     }
 
     parameter_meta {
         coverage_labels: "An array of equal length to `coverage_beds_len` which determines the prefix label applied to coverage output files. If an empty array is supplied, defaults of `regions1`, `regions2`, etc. will be used."
-        input_molecule: "Must be `DNA` or `RNA`"
-        gtf_provided: "Was a GTF supplied by the user? Must be `true` if `input_molecule = RNA`."
+        rna: "Is the input molecule RNA?"
```

Comment:
What about `input` -> `sequenced`? I think that's more accurate. It is input to this workflow though, but the molecule is what was sequenced.